scripts/loki_transform.py

```diff
@@ -113,12 +114,15 @@ def cli(debug):
               help="Recursively derive explicit shape dimension for argument arrays")
 @click.option('--eliminate-dead-code/--no-eliminate-dead-code', default=True,
               help='Perform dead code elimination, where unreachable branches are trimmed from the code.')
+@click.option('--log-level', '-l', default='info', envvar='LOKI_LOGGING',
```

Comment:
I just noticed that the log level could already be specified via this environment variable: https://github.com/ecmwf-ifs/loki/blob/1d95896f118ede6ada0e9bdd3e38921e81f3a8a9/loki/__init__.py#L49-L50
Not sure if having this twice might cause any problems?

Reply:
It doesn't cause problems _per se_, as the CLI version takes precedence (the naming alias is no coincidence). However, I still think exposing an explicit CLI option is a neat thing to have and having two ways of specifying env flags is iffy. The only remaining use case for this is now toggling log-level for the pytest test base invocation - I'm not sure if this is sufficiently useful to warrant having two ways of doing this?

Reply:
In principle we could also provide an equivalent log level overwrite switch for pytest specifically:
https://docs.pytest.org/en/stable/reference/reference.html#pytest.hookspec.pytest_addoption